cli/patch.go

```diff
@@ -489,22 +491,40 @@ func (lc *ListCommand) listProjects() error {
 	return errors.WithStack(w.Flush())
 }
 
-func (lc *ListCommand) listDistros() error {
+func (lc *ListCommand) listDistros(onlyUserSpawnable bool) error {
 	ctx := context.Background()
-	ac, _, _, err := getAPIClients(ctx, lc.GlobalOpts)
-	if err != nil {
-		return errors.WithStack(err)
+	client, settings, client_err := getAPIV2Client(ctx, lc.GlobalOpts)
+	if client_err != nil {
+		return errors.WithStack(client_err)
 	}
-	notifyUserUpdate(ac)
+	client.SetAPIUser(settings.User)
```

Comment:
Since the distros route doesn't require an authenticated user, this is unnecessary